packages/node-common/package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "@cursorless/file-system-common",
+  "name": "@cursorless/node-common",
```

Comment:
I am tempted to rename this package. Our convention has been that things with `-common` don't import `cursorless-engine`, so that they can be imported in any context, eg test harness. I don't feel strongly though and I can't come up with a better name

Reply:
I don't have a better na?me either. We can always rename it in the future if we come up with a better one?